easybuild/easyconfigs/l/LncLOOM/LncLOOM-2.0-foss-2020b.eb

```diff
@@ -0,0 +1,76 @@
+easyblock = 'PythonBundle'
+
+name = 'LncLOOM'
+version = '2.0'
+
+homepage = 'https://github.com/lncLOOM/lncLOOMv2/'
+description = """LncLOOM is a graph-based framework that uses integer
+programming to identify combinations of short motifs that are deeply conserved
+in rapidly evolving sequences."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('MAFFT', '7.475', '-with-extensions'),
+    ('BLAST+', '2.11.0'),
+    ('Gurobi', '9.1.0', '', True),
+    ('Biopython', '1.78'),
+]
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+exts_list = [
+    ('amply', '0.1.4', {
+        'checksums': ['cb12dcb49d16b168c02be128a1527ecde50211e4bd94af76ff4e67707f5a2d38'],
+    }),
+    ('networkx', '2.5', {
+        'checksums': ['7978955423fbc9639c10498878be59caf99b44dc304c2286162fd24b458c1602'],
+    }),
+    ('PuLP', '2.4', {
+        'source_tmpl': '%(version)s.tar.gz',
+        'source_urls': ['https://github.com/coin-or/pulp/archive/'],
+        'checksums': ['d52d52b6078ea2c503b33814738866902d661cfe60d7a76251d50423e4af3ad0'],
+    }),
+    ('gurobipy', '9.1.0', {
```

Comment:
Please use a proper dependency instead for this, see the existing `Gurobi-9.0.0-intel-2018b-Python-3.6.6.eb` easyconfig (which needs an update, of course)

Reply:
Oh, there is one? I found the solver itself, but `gurobipy` isn't anywhere in here which confused me. There is a problem though - if I update the package to 2020b and use the newest Python, I'll need to remove the version suffix, which will then conflict with the binary solver itself (as opposed to gorubipy, the python interface). @boegel

Reply:
Or maybe one Gorubi with the SYSTEM toolchain (the solver) and one with a Python toolchain (foss/intel - for the Python interface) will be alright? Or maybe merge the two packages? But then the solver wouldn't be able to use the system toolchain, I think... Not sure of the best course of action here.

Reply:
There will be no conflict: the one with the system toolchain is the solver, the one with a non-system toolchain and Python dependency is the Python bindings. It could make sense to rename the latter to `gurobipy` though, if that's a standard name (is it?).

Reply:
"Gurobi comes with a Python extension module called “gurobipy” that offers convenient object-oriented modeling constructs and an API to all Gurobi features.", on pypi it is gurobipy, and overall I see a lot of mentions to gurobipy, so I'd say the standard name for the Python wrapper really is gurobipy

Reply:
I take that, the custom easyblock confused me - there is Gurobi+SYSTEM which is only the solver, and there is Gurobi+any other toolchain, which is the solver + gurobipy - the Python part being optional in the easyblock.

Reply:
Ah, indeed. Then it makes sense to depend on a Gurobi installation that includes the Python bindings here?